springboottrades-quotes/src/main/java/io/pivotal/quotes/controller/QuoteV1Controller.java

```diff
@@ -1,137 +0,0 @@
-package io.pivotal.quotes.controller;
-
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
-import javax.servlet.http.HttpServletResponse;
-
-import io.pivotal.quotes.domain.CompanyInfo;
-import io.pivotal.quotes.domain.Quote;
-import io.pivotal.quotes.exception.SymbolNotFoundException;
-import io.pivotal.quotes.service.QuoteService;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpHeaders;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.ExceptionHandler;
-import org.springframework.web.bind.annotation.MatrixVariable;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-
-/**
- * Rest Controller providing the REST API for the Quote Service. Provides two
- * calls (both HTTP GET methods): - /quote/{symbol} - Retrieves the current
- * quote for a given symbol. - /company/{name} - Retrieves a list of company
- * information for companies that match the {name}.
- * 
- * @author David Ferreira Pinto
- *
- */
-@RestController
-@RequestMapping(value = "/v1")
```

Comment:
why remove the v1 from the path? its a good thing to have as we can then have multiple versions of the API running.